test/plugin/test_out_file.rb

```diff
@@ -37,6 +37,24 @@ def create_driver(conf = CONFIG)
       assert_equal :gzip, d.instance.instance_eval{ @compress_method }
     end
 
+    test 'using root_dir for buffer path' do
```

Comment:
It's just configuration error for `config_param :path` (in `out_file`), so it's clear that `out_file` raises configuration error for missing path...
But anyway, i added it.